packages/flutter/lib/src/foundation/print.dart

```diff
@@ -172,3 +172,11 @@ Iterable<String> debugWordWrap(String message, int width, { String wrapIndent: '
     }
   }
 }
+
+/// Returns a 5 character long hexadecimal hash code for [object].
```

Comment:
Can you elaborate on these docs, for example saying that it uses `Object.hashCode`'s twenty least-significant bits?
Also maybe rather than saying it returns a hash code, say that it returns a string based on the hash code? Hash codes are integers and this returns a string. Feel free to phrase this however you want.